docs/Documentation/Drivers/Image/RvicParams.md

```diff
@@ -0,0 +1,7 @@
+﻿# RvicParams
```

Comment:
Rename this file / heading Routing something similar. Then add a section that describes the default stub routing model. Then add a section that points to the RVIC documentation and describes how to configure VIC with an RVIC parameter file.  Users will need to know a few things 1) the python version of RVIC should be used to generate the parameters, 2) the domains must match, and 3) where to find the RVIC documentation. You have some of that below.